.github/workflows/api-code-gen.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - run: make api-code-gen
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          env:
+            ENABLE_GIT_PUSH: true
```

Comment:
optional: semantically this makes more sense to me as a cli arg rather than an env var.